package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "packageManager": "yarn@1.22.19",
```

Comment:
```suggestion
```

We use [Volta](https://volta.sh/) to manage our package manager and it's version. We should probably add this to our contributing docs!